### ISSUE TITLE
Mouthwash kit receipt error in Biospecimen Dashboard

### ISSUE THREAD

**Issue by erincschwartz** [episphere/connect#981]
FNL reported the following error when attempting to receipt mouthwash kit, Collection ID CHA000022 0007 in the Biospecimen Dashboard

![Image](https://github.com/episphere/connect/assets/157617486/cc43653a-86e0-49c2-a5a6-1c1f3c894874)

Prod data showed the tracking number as

![Image](https://github.com/episphere/connect/assets/157617486/e76c1c3c-cb03-4314-94ae-647b377ce59e)

FNL reports that they did not receive any error messages after scanning the kit tracking number, they received the checkmark in the Dashboard. 

@JoeArmani suggested FNL attempt to manually enter the tracking number with the parenthesis. FNL was able to receipt the kit and all receipt data populated on the back end.

**Comment by sonyekere** [comment]
Hi @erincschwartz is this also connected to issue #970 ?

**Comment by erincschwartz** [comment]
@sonyekere I do not believe this is directly related to https://github.com/episphere/connect/issues/970. @JoeArmani, thoughts?

**Comment by JoeArmani** [comment]
@sonyekere @erincschwartz I agree that it's a separate issue. It would probably be most efficient for a single developer to tackle the range of Home Collections issues for familiarity and also to avoid merge conflicts (if they're all resolved in the upcoming cycle).

**Comment by sonyekere** [comment]
Thanks Both. @JoeArmani good point!

**Comment by JoeArmani** [comment]
@amber-emmes @erincschwartz Erin, please confirm/update this:
1. The issue starts with the scan in 'Assemble Kits.' We need to eliminate this possibility throughout the process.
2. Please ensure all the tracking number fields allow numeric input only. I'd mock the returned data from the scanner to make sure the process works as expected -- they will be using a scanner in real-time (not manual entry).
3. We'll want to update those records in prod. There are only 4 records with this issue currently. @erincschwartz should this go into the prod changelog or are you expecting many more in the upcoming weeks?

**Comment by amber-emmes** [comment]
A PR to fix this going forward has been submitted; however, it will not fix the existing records in prod, which must be fixed separately.

**Comment by erincschwartz** [comment]
We will request a prod change for existing records to coincide with the Prod release, as there may be more cases in the meantime if FNL team forgets to remove the parenthesis. That way, will not have to make multiple prod change requests.

**Comment by amber-emmes** [comment]
This is now in dev and ready for testing.

**Comment by wyattks** [comment]
Parenthesis automatically were removed when scanned in for the kit.

Tested in Dev.  Ready for Stage.

**Comment by wyattks** [comment]
Tested in Stage. Ready for Prod.

**Comment by amber-emmes** [comment]
This is now in prod.
